code.gs

```diff
@@ -445,12 +445,8 @@ MergedContact.prototype.getInfoFromGPlus = function (gPlusProfileId) {
   log.add('Extracting info from Google Plus Profile...', Priority.INFO);
 
   // Profile ID.
-  try {
-    gPlusProfile = Plus.People.get(gPlusProfileId);
-    if (gPlusProfile === null) {
-      throw new Error('');
-    }
-  } catch (err) {
+  gPlusProfile = (Plus ? Plus.People.get(gPlusProfileId) : null);
```

Comment:
This will not work since `Plus.People.get(gPlusProfileId)` throws a 404 error when `gPlusProfileId` is not valid, causing the whole script to crash instead of simply aborting the execution of `MergedContact.prototype.getInfoFromGPlus()`.

I guess you saw the `if (gPlusProfile === null)` and thought that was what the function returned in case of an error; at least this is what I immediately thought reading this code _right now._  
I say _right now_ because I was the one who wrote those exact lines 7 months ago (ff8d89012906be42527ec2122c01b5553ae7aaf9), but, for the life of me, I cannot remember why would I've had to put that silly if condition that will never be met.

Looking at the documentation I see no way for that API call to return `null`, so that `if` clause seems to be overkill and misleading at the same time.  
I am running some tests to verify this: will report back as soon as they are done.